### DISCUSSION TITLE
How to restore linked messenger contacts?

### DISCUSSION THREAD

**Discussion by someone2037492034** [FossifyOrg/General-Discussion#675] [General]
Hi, for example, if I have the same phone number/contact in Signal, then it shows in Fossify Phone and Contacts, but if I click on the contact, click on Signal, and delete it, then how do I restore it? I think it is called app contacts sync or something? I deleted all my contacts and now I can't access the shortcut...

**Comment by Aga-C** [comment]
> I deleted all my contacts and now I can't access the shortcut...

Have you deleted all contacts from Signal, or have you removed all contacts from your phone storage?

**Comment by someone2037492034** [comment]
What I did was 1. create a backup 2. I deleted all contacts from all places that they were, just deleted until none were left. 3. restored the backup, but the Signal shortcut thing was missing... Is it possible to restore it? I didn't delete them from Signal.

**Reply by Aga-C** [reply]
Unfortunately, there's nothing to do on Fossify Contacts side. Signal would need to recreate the contact list. 

You can try to do this in Signal, but I don't know if it will help: tap creating a new conversation, and at the bottom there is Refresh contacts. If it doesn't help, try to look for answers on Signal's community.

**Comment by someone2037492034** [comment]
Reinstalling Signal completely helped.
